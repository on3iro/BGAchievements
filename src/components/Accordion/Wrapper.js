import styled from 'styled-components'

const Wrapper = styled.div`
  width: 100%;
  height: 90%;

  overflow: auto;

  div:last-child {
    margin-bottom: 0;
  }
`

export default Wrapper
