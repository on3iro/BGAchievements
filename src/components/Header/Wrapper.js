/**
  * Styled container for Header
  *
  *
  * @namespace Header.Wrapper
  * @memberOf Header
  */

import styled from 'styled-components'

const Wrapper = styled.header`
  position: fixed;
  top: 0px;
  margin-bottom: 0;

  min-height: 64px;
  height: 64px;
  width: 100%;

  background-color: ${props => props.theme.mainBG};
  color: ${props => props.theme.noneWhite};

  @media (max-width: 900px) {
    position: static;

    height: auto;
  }
`

export default Wrapper
