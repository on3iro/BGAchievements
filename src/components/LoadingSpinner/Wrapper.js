import styled from 'styled-components'

const Wrapper = styled.div`
  width: 100%;
  height: 100%;
  text-align: center;
`

export default Wrapper
