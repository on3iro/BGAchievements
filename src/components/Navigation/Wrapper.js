import styled from 'styled-components'

export default styled.nav`
  margin: 12px 50px 15px 0;
  height: 100%;

  text-align: right;
`
