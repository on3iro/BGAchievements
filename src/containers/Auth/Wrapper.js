/**
  * Renders a styled div for the PasswordGenerator
  *
  * @namespace PasswordGenerator.Wrapper
  * @memberOf PasswordGenerator
  */

import styled from 'styled-components'

const Wrapper = styled.section`
  margin-top: 20px;
`

export default Wrapper
