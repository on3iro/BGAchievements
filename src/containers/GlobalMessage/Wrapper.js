import styled from 'styled-components'

const Wrapper = styled.div`
  margin: 80px 0 0 0;
  padding: 10px 20px;
  height: 40px;

  border-radius: 4px;

  background: ${props => props.error ? props.theme.warningHover : props.theme.primaryHover};
  opacity: ${props => props.empty ? '0' : '.6'};
  color: #fff;
`

export default Wrapper
