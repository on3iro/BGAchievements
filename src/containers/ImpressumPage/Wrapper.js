import styled from 'styled-components'

const Wrapper = styled.section`
  width: 100%;
  height: 100%;

  overflow: auto;
`

export default Wrapper
