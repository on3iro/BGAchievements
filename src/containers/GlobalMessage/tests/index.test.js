import React from 'react'
import { shallow } from 'enzyme'

import Wrapper from '../Wrapper'
import { GlobalMessage, mapStateToProps } from '../index'

describe('<GlobalMessage />', () => {
  it('should render <Wrapper />', () => {
    const wrapper = shallow(<GlobalMessage />)
    expect(wrapper.find(Wrapper).exists()).toBe(true)
  })

  it('should render <Wrapper /> with empty prop if no error or message is present', () => {
    const wrapper = shallow(<GlobalMessage />)
    expect(wrapper.find(Wrapper).props().empty).toBe(true)
  })

  it('should render <span /> with error', () => {
    const wrapper = shallow(<GlobalMessage error='Error' />)
    expect(wrapper.find('span').exists()).toBe(true)
  })

  it('should render <span /> with message', () => {
    const wrapper = shallow(<GlobalMessage message='Messag' />)
    expect(wrapper.find('span').exists()).toBe(true)
  })

  describe('react-redux', () => {
    it('should map state to props', () => {
      expect(mapStateToProps({
        GlobalMessages: {
          GlobalMessage: 'hi',
          GlobalError: 'Error'
        }
      })).toMatchSnapshot()
    })
  })
})
