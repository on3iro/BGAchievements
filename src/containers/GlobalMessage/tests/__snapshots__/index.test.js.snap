exports[`<GlobalMessage /> react-redux should map state to props 1`] = `
Object {
  "error": "Error",
  "message": "hi",
}
`
